types/react-instantsearch-core/index.d.ts

```diff
@@ -771,5 +771,13 @@ export function connectHitInsights(
 ) => React.ComponentType<Omit<ConnectHitInsightsProvided, { insights: WrappedInsightsClient }>>;
 export function connectVoiceSearch(Composed: React.ComponentType<any>): React.ComponentClass<any>;
 
+export interface DynamicWidgetsProps  {
+    children: React.ReactNode;
+    attributesToRender: string[];
+    fallbackComponent?: React.ComponentType<{ attribute: string }>;
+}
+
+export function connectDynamicWidgets(Composed: React.ComponentType<any>): React.ComponentClass<any>;
+
```

Comment:
the DynamicWidgets component doesn't seem to be exported from here, and I guess also missing in the new tests, thanks if you could add those!

Reply:
Could you please help me? I am not sure how I have to update this PR.

Reply:
in the test file we need to add something like

```
import { DynamicWidgets } from 'react-instantsearch-core';

<DynamicWidgets fallbackComponent={RefinementList}><RefinementList attribute="brand"/></DynamicWidgets>
```


in the dom test:

```
import { DynamicWidgets } from 'react-instantsearch-dom';

<DynamicWidgets fallbackComponent={RefinementList} className="test"><RefinementList attribute="brand"/></DynamicWidgets>
```